runtime/lua/vim/lsp/protocol.lua

```diff
@@ -1,9 +1,16 @@
 --- @diagnostic disable: duplicate-doc-alias
 
+-- TODO(clason) can be simplified after reverse lookup is removed
 local function get_value_set(t)
```

Comment:
```suggestion
-- TODO(clason) can be simplified after reverse lookup is removed
---@param t table<any, any>
---@return number[]
local function get_value_set(t)
```

A plain loop variant would also work:

```
local function get_value_set(t)
  local result = {}
  for _, v in pairs(t) do
    if type(v) == "number" then
      table.insert(result, v)
    end
  end
  return result
end
```

But I guess either is fine.